### PR TITLE
update links and install instructions

### PR DESCRIPTION
**Pull request by jessegoodier**
## What does this PR change?

*fixes some kubecost>opencost naming in files
*update links to opencost website

## Does this PR relate to any other PRs?
* https://github.com/kubecost/opencost-website/pull/28

## How will this PR impact users?
* Easier to install opencost and find sample queries.

## Does this PR address any GitHub or Zendesk issues?
No

## How was this PR tested?
* eks/gcp clusters

## Does this PR require changes to documentation?
* done in https://github.com/kubecost/opencost-website/pull/28

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Opencost release? If not, why not?
* NA, just updating docs

### CODE REVIEW

**Review by mbolt35 — APPROVED** [review]
Since this is somewhat of a duplicate of https://github.com/opencost/opencost/pull/1280 and that was approved, doing same here :)

**Comment by jessegoodier** [comment]
Could someone please merge this? Links are broken from the new docs on opencost-website

**Comment by michaelmdresser** [comment]
Trusting Bolt's approval here :grin:

File: docs/opencost.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
+{
+	"info": {
+		"_postman_id": "bba454a2-641d-4481-9ccd-0395a6054a20",
```
Comment (dwbrown2) [inline docs/opencost.postman_collection.json:3]: @jessegoodier are there other APIs we want to document here?

**Review by dwbrown2 — COMMENTED** [review]
Great stuff!! Huge thanks @jessegoodier and @michaelmdresser!

**Pull request merged**